nilearn/image/image.py

```diff
@@ -502,6 +502,51 @@ def swap_img_hemispheres(img):
 
     # create swapped nifti object
     out_img = nibabel.Nifti1Image(img.get_data()[::-1], img.get_affine(),
-        header=img.get_header())
+                                  header=img.get_header())
 
     return out_img
+
+
+def index_img(imgs, index, check_imgs=True):
+    """Indexes into a 4D Niimg-like object in the fourth dimension
+
+    Parameter
+    ---------
+    imgs: 4D Niimg-like object
+          See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
+
+    index: Any type compatible with numpy array indexing
+           Used for indexing the 4D data array in the fourth dimension
+
+    check_imgs: bool
+                Whether to check that `imgs` is a 4D Niimg-like object
+
+    Returns
+    -------
+    output: nibabel.Nifti1IMage
+            4D image whose data is a subset of the original image data
+
+    """
+    if check_imgs:
+        imgs = check_niimgs(imgs)
+
+    return nibabel.Nifti1Image(imgs.get_data()[:, :, :, index],
```

Comment:
I have to admit I just copied and pasted @GaelVaroquaux's code. From a quick grep, we do tend to use `...` more often than `:, :, :`. `...` also looks slightly easier on the eyes in my biased opinion.